include/bluetooth/mesh/lightness_srv.rst

```diff
@@ -8,58 +8,54 @@ It should be instantiated in the light fixture node.
 
 The Light Lightness Server adds the following new model instances in the composition data, in addition to the extended :ref:`bt_mesh_lvl_srv_readme` and :ref:`bt_mesh_ponoff_srv_readme` models:
 
-- Light Lightness Server
-- Light Lightness Setup Server
+* Light Lightness Server
+* Light Lightness Setup Server
 
 These model instances share the states of Light Lightness Server, but accept different messages.
-This allows fine-grained control of the access rights for the Light Lightness states, as these model instances can be bound to different application keys.
+This allows for a fine-grained control of the access rights for the Light Lightness states, as these model instances can be bound to different application keys:
 
-The Light Lightness Server only provides write access to the Light state for the user, in addition to read access to all the meta states.
-
-The Light Lightness Setup Server provides write access to the Default Light and Light Range meta states.
-This allows the configurator devices to set up the range and the default value for the Light state.
+* The Light Lightness Server only provides write access to the Light state for the user, in addition to read access to all the meta states.
+* The Light Lightness Setup Server provides write access to the Default Light and Light Range meta states.
+  This allows the configurator devices to set up the range and the default value for the Light state.
 
 States
 ======
```

Comment:
This section is missing the common "this model contains the following states" text